src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
     public class ConsoleLogger : BaseLogger, IAnalysisLogger
```

Comment:
>ConsoleLogger [](start = 17, length = 13)

See src\Sarif.Driver\DriverExtensionMethods.cs 'ValidateOutputOptions' for where we're at with arg validation. My concern: if we pass --quiet and have no --output arg, that's bad.

Reply:
Done.

---
In reply to: [559736856](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559736856) [](ancestors = 559736856)